testnet/stacks-signer/src/signing_round.rs

```diff
@@ -49,20 +49,51 @@ impl StateMachine for SigningRound {
 #[derive(Serialize, Deserialize, Debug)]
 pub enum MessageTypes {
     DkgBegin(DkgBegin),
-    DkgEnd,
-    SignatureShare(SignatureShare),
+    DkgEnd(DkgEnd),
+    DkgQuery,
+    DkgPublicShares(DkgShares),
+    DkgPrivateShares(DkgShares),
+    NonceRequest,
+    NonceResponse(NonceResponse),
+    SignShareRequest(SignatureShareRequest),
+    SignShareResponse(SignatureShareResponse),
```

Comment:
@donpdonp @xoloki I could use your help in these message variants. I'll read up more on Frost and the devnet design to ensure I find the appropriate representations here, but any pointers are greatly appreciated.

Reply:
Pretty sure that dkg public pyruvate shares can't share a dkgshares type.  The public share is a polynomial commitment, private is a set of tuples of (uint, Scslar) which is the poly evaluated at the int

Reply:
Looking now